app/views/candidate_interface/content/providers.html.erb

```diff
@@ -1,15 +1,17 @@
 <% content_for :title, t('page_titles.providers') %>
 
-<h1 class="govuk-heading-xl govuk-!-width-two-thirds">
-  <%= t('page_titles.providers') %>
-</h1>
+<% cache 'courses-grouped-by-region-and-provider', expires_in: 15.minutes do %>
```

Comment:
going to have a look in Kibana and see how often this page is requested

Reply:
Had a quick look at Kibana and it looks like there were 278 hits in the past 7 days. At busy times its getting hit about 10 times an hour so a 15 minute cache would speed up a fair proportion of requests and a majority if we leave it running after the 2022 cycle begins when load increases.

<img width="976" alt="image" src="https://user-images.githubusercontent.com/450843/131649665-4a36046f-3c14-4b7e-a7f9-b9eab1d445eb.png">